### ISSUE TITLE
'make tests' error

### ISSUE THREAD

**Issue by AmeBel** [opencog/atomspace#183]
```
In file included from /atomspace/build/tests/atomspace/TVMergeUTest.cpp:17:0:
/atomspace/tests/atomspace/TVMergeUTest.cxxtest: In member function 'void TVMergeUTest::testIndefiniteMerge()':
/atomspace/tests/atomspace/TVMergeUTest.cxxtest:142:69: error: 'TS_SKIP' was not declared in this scope
         TS_SKIP("Merging with IndefiniteTruthValue not implemented!");
                                                                     ^
make[3]: *** [tests/atomspace/CMakeFiles/TVMergeUTest.dir/TVMergeUTest.cpp.o] Error 1
make[2]: *** [tests/atomspace/CMakeFiles/TVMergeUTest.dir/all] Error 2
make[1]: *** [CMakeFiles/tests.dir/rule] Error 2
make: *** [tests] Error 2
```

**Comment by jswiergo** [comment]
The macro TS_SKIP is available since version cxxtest 4.2 (2013-03-17). The recent version is cxxtest 4.4.
I guess you has found this problem only for Travis CI tests. The macro is convenient for programmers so I think it would be better to upgrade cxxtest version on Travis if it is not a big problem. Are you able to upgrade?

**Comment by AmeBel** [comment]
Thanks, using 4.3 version for now.
